### DISCUSSION TITLE
Statistic Settins in configuration

### DISCUSSION THREAD

**Discussion by ei8ht187** [coinfoundry/miningcore#1168] [Q&A] [answered]
Hello again, 
today I've found following configuration options for Miningcore: 
    "statistics": {
        "updateInterval": 120,
        "hashrateCalculationWindow": 10,
        "gcInterval": 4,
        "cleanupDays": 180
        }

What will be cleaned during gcInterval and cleanupDays?

Kind regards

ei8ht

**Comment by oliverw** [comment, marked as answer]
The last N `cleanupDays` of statistics for pools and miners will be deleted.

**Reply by ei8ht187** [reply]
Again thanks for your explanation

**Reply by oliverw** [reply]
Any time 😄
